src/App.js

```diff
@@ -0,0 +1,27 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          <code>src/App.js</code>
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
```

Comment:
- [ ] As per the project requirements, you are required to delete all the boilerplate text, images, and styles that the CRA created.

![image](https://user-images.githubusercontent.com/86318284/189194828-4803a7d7-47dc-4306-a572-854f21461f08.png)

- [ ] Also, Kindly note that the project requirements require you to now import the <Calculator/> component into the <App/> component and not in the index.js file as you have done. Kindly fix this.

![image](https://user-images.githubusercontent.com/86318284/189195126-eeba4d42-2efe-4974-97e8-0c20b52ca4e4.png)

Reply:
I resolved the issue, thank you @Nemwel-Boniface